lib/constructor/curated_attachments.lua

```diff
@@ -1566,7 +1594,11 @@ local curated_attachments <const> = {
                               hash = 3663644634,
                               name = "Penumbra FF",
                               type = "VEHICLE"
-                          }, {
+                          },{
+                              hash = 1076201208,
+                              name = "300R",
+                              type = "VEHICLE"
+                          } {

```

Comment:
missing a comma here?

Reply:
I dont think so? that is what it said the display name was

Reply:
I mean between the objects =) `} {` should be `}, {`